rq/worker.py

```diff
@@ -1160,3 +1164,22 @@ def request_force_stop_sigrtmin(self, signum, frame):
         info = dict((attr, getattr(frame, attr)) for attr in self.frame_properties)
         self.log.warning('raising ShutDownImminentException to cancel job...')
         raise ShutDownImminentException('shut down imminent (signal: %s)' % signal_name(signum), info)
+
+
+class RoundRobinWorker(Worker):
+    """
+    Modified version of Worker that dequeues jobs from the queues using a round-robin strategy.
+    """
+    def reorder_queues(self, reference_queue):
+        pos = self.queues.index(reference_queue)
+        self.queues=self.queues[pos+1:]+self.queues[:pos+1]
```

Comment:
Can you add the proper spacing around `=` and `+`?

Reply:
done